src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
It would be good to clear whatever we use local storage for that should not be persisted between users... Nothing afaik... On login, that would be good. We cannot consider potential security concerns as edge cases so thank you for reviewing this, it should be very easy to do what you suggest if you can make that change?